extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -212,6 +213,9 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         isInManualExperiment(this.element);
     const width = Number(this.element.getAttribute('width'));
     const height = Number(this.element.getAttribute('height'));
+
+    validateAdSenseAdHeight(height);
```

Comment:
Shouldn't we only do this check if it's a responsive ad?

Reply:
I was thinking the same, will add the check.